mozilla_pipeline_schemas/bigquery.py

```diff
@@ -1,26 +1,34 @@
+import argparse
 import json
 import os
 import shutil
-import tempfile
-import argparse
```

Comment:
hm, wouldn't `black`  have found this? Can you run with black?

Reply:
`black` doesn't change imports, that's the job of `isort`.